scripts/rebuild_languages.py

```diff
@@ -9,7 +10,8 @@
 from urllib.request import urlopen
 
 
-def prettier(path):
+def prettier(path, repoDir):
```

Comment:
If you leave `repoDir` as a global var, this chnge isn't necessary.

Suggest to leave all constants as global vars, I think this is about:
- repoDir
- langDir
- localizationJSPath
- languagesBlock, but rename to languagesBlockTemplate, and use `languagesBlockTemplate.format(...)`
- languageSourceTemplate

Reply:
Thanks. Done.